seed/tasks.py

```diff
@@ -211,6 +229,67 @@ def delete_organization_inventory(org_pk, prog_key=None, chunk_size=100, *args,
     return progress_data.result()
 
 
+@shared_task
+@lock_and_track
+def delete_organization_column(column_pk, org_pk, prog_key=None, chunk_size=100, *args, **kwargs):
+    """Deletes an extra_data column from all merged property/taxlot states."""
+
+    column = Column.objects.get(id=column_pk, organization_id=org_pk)
+
+    progress_data = ProgressData.from_key(prog_key) if prog_key else ProgressData(
+        func_name='delete_organization_column', unique_id=column_pk)
+
+    ids = []
+
+    if column.table_name == 'PropertyState':
+        ids = list(
+            PropertyState.objects.filter(organization_id=org_pk, data_state=DATA_STATE_MATCHING,
+                                         extra_data__has_key=column.column_name).values_list('id', flat=True)
+        )
+    elif column.table_name == 'TaxLotState':
+        ids = list(
+            TaxLotState.objects.filter(organization_id=org_pk, data_state=DATA_STATE_MATCHING,
+                                       extra_data__has_key=column.column_name).values_list('id', flat=True)
+        )
+
+    total = len(ids)
+
+    # total is the number of records divided by the chunk size
+    progress_data.total = total / float(chunk_size)
+    progress_data.data['completed_records'] = 0
+    progress_data.data['total_records'] = total
```

Comment:
see other comment about adding these fields to the `initialize` method for ProgressData